ci-operator/step-registry/openshift/ci-security/snyk/scan/openshift-ci-security-snyk-scan-commands.sh

```diff
@@ -13,6 +13,13 @@ chmod +x ${SNYK_DIR}/snyk
 echo snyk installed to ${SNYK_DIR}
 ${SNYK_DIR}/snyk --version
 
+# install jq if not installed
+if ! [ -x "$(command -v jq)" ]; then
+    curl -Lo ${SNYK_DIR}/jq "https://github.com/jqlang/jq/releases/download/jq-1.7/jq-linux-amd64"
```

Comment:
For what it's worth, executing pre-built binaries downloaded from the net without knowing how/where they were built, if they were tampered with, ... could be problematic, especially when it's related to security.

Reply:
@cfergeau good point. I added a check for the checksum. Or any better approaches?

Reply:
Thanks! If this runs on fedora/RHEL, you can also `yum install jq`.